server/app.js

```diff
@@ -6,44 +6,52 @@ import webpackMiddleware from 'webpack-dev-middleware';
 import webpackHotMiddleware from 'webpack-hot-middleware';
 import webpackConfig from '../webpack.config';
 
-
 import userRoute from './route/apiRoute';
 
-
 const app = express();
 
 const compiler = webpack(webpackConfig);
 
-app.use(webpackMiddleware(compiler, {
-  hot: true,
-  publicpath: webpackConfig.output.publicPath,
-  stats: { colors: true },
-  noInfo: true,
-}));
+app.use(
+  webpackMiddleware(compiler, {
+    hot: true,
+    publicpath: webpackConfig.output.publicPath,
+    stats: { colors: true },
+    noInfo: true
+  })
+);
 app.use(express.static(path.join(__dirname, '../template/Public')));
 app.use(webpackHotMiddleware(compiler));
 
 app.use(bodyParser.json());
```

Comment:
Check out the express documentation. For express version 4~, you no longer need the bodyparser module as it is embedded in express as a middleware. Thus, `app.use(express.json())` and `app.use(express.urlencoded)`...